docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  todoapp:
+    image: todoapp:1.0.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - todoapp-network
+    depends_on:
+      - mysql-db
+    restart: unless-stopped
+
+  mysql-db:
+    image: mysql-local:1.0.0
```

Comment:
The `image` field specifies `mysql-local:1.0.0`. Ensure that this image is available locally or in a Docker registry. If you intend to build the image from the Dockerfile.mysql, you might not need this field.